templates/collections/time_period_explorer_index_page.html

```diff
@@ -5,16 +5,16 @@
 {% block content %}
     {% include 'includes/generic-intro--dark.html' %}
 
-    <div class="container">
-        <div class="row">
-            <ul class="row card-group">
+    <div class="container mt-4">
+            <h2 class="sr-only">Select a time period</h2>
+            <ul class="row card-group--list-style-none">
```

Comment:
Similar to comments above, we should avoid mixing BEM and Bootstrap on the same element.